components/Utils/ActionBubble.js

```diff
@@ -103,7 +103,8 @@ const ActionBubble = ({ addresses, bounty, action }) => {
   }
 
   if (action?.receiveTime || action?.refundTime) {
-    const funder = senderEnsName || shortenAddress(action.sender.id);
+    const refunder = bounty.deposits.filter((deposit) => deposit.id === action.depositId)[0]?.sender?.id;
+    const funder = senderEnsName || shortenAddress(action.sender?.id) || shortenAddress(refunder);
```

Comment:
Just wondering, why does it have an option to use the refunder address as the funder? Isn't the action.sender.id good enough?

Reply:
action.sender.id was breaking for refunds because it wasn't mapped correctly in the Graph - that's why I used the workaround to get the sender.id for refunds regardless. But this Subgraph PR should fix this: https://github.com/OpenQDev/OpenQ-Graph/pull/5

Reply:
Ah, I see